test/withStyles_test.jsx

```diff
@@ -59,7 +61,7 @@ describe('withStyles()', () => {
         return null;
       }
 
-      const WrappedComponent = withStyles(() => ({}))(MyComponent);
+      const WrappedComponent = withStyles(() => ({}))(safeSFC(MyComponent));
```

Comment:
I think this is an interesting solution, but I think it actually makes the tests less readable. My preference would be to not include this at all, and just not run tests against React 0.13.